### ISSUE TITLE
Undocumented exceptions thrown by System.Windows.Media.ColorConverter.ConvertFromString()

### ISSUE THREAD

**Issue by ChrisMaddock** [dotnet/docs#4940]
https://docs.microsoft.com/en-gb/dotnet/api/system.windows.media.colorconverter.convertfromstring?view=netframework-4.7.1#System_Windows_Media_ColorConverter_ConvertFromString_System_String_

The `ConvertFromString(String)` method isn't documented as throwing any exceptions, but at least throws a `FormatException` for invalid input. I haven't done a PR, as I don't know what exceptions are thrown by any overloads etc.

**Comment by mairaw** [comment]
Thanks @ChrisMaddock. You can see the source for that at https://referencesource.microsoft.com/#System/compmod/system/componentmodel/TypeConverter.cs. We're adding this to our backlog but feel free to jump in.

**Comment by Niladri24dutta** [comment]
it throws `NotSupportedException` when the input string is `null` or if there are any unsupported values in the string according to the line below https://referencesource.microsoft.com/#System/compmod/system/componentmodel/TypeConverter.cs,256 , the other overloads also seems to throw the same exception. But it isn't mentioned in the docs . Should I create a PR for this?

**Comment by rpetrusha** [comment]
If you'd like to create the PR, @Niladri24dutta, we'll be delighted to review it.

**Comment by Niladri24dutta** [comment]
@rpetrusha ok I will create a PR for this, is there any existing format to show the exception for any method in the docs ? Should I follow any reference?

**Comment by rpetrusha** [comment]
Yes, there is an existing format, @Niladri24dutta. It uses XML. The following is an example that illustrates the \<exception> element, as well as the two elements  (\<paramref /> and \<see langword />) most commonly embedded in text:
```
<exception cref="T:System.ArgumentNullException">
          <paramref name="array" /> is <see langword="null" />.</exception>
```

There is an \<exception> element for each exception, and the set of \<exception> elements follows the \<remarks> element. 
The documentation is in the dotnet/dotnet-api-docs repo in the xml folder.
Let me know if you have any questions.
